webapp/components/topbar/SwitchPageButton.js

```diff
@@ -2,18 +2,25 @@ import React, { Component } from 'react';
 
 import { withStyles } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
-import IconButton from '@material-ui/core/IconButton';
 import { Tooltip } from 'netpyne/components';
```

Comment:
- Tooltip import is unused & can be removed
- same for `simulateNetwork` and `disableSimulate` in `render` method